Rubberduck.Parsing/Preprocessing/VBAPreprocessorVisitor.cs

```diff
@@ -18,26 +18,14 @@ public sealed class VBAPreprocessorVisitor : VBAConditionalCompilationParserBase
             ICharStream stream,
             CommonTokenStream tokenStream)
         {
-            if (stream == null)
-            {
-                throw new ArgumentNullException(nameof(stream));
-            }
-            if (tokenStream == null)
-            {
-                throw new ArgumentNullException(nameof(tokenStream));
-            }
-            if (symbolTable == null)
-            {
-                throw new ArgumentNullException(nameof(symbolTable));
-            }
             if (predefinedConstants == null)
             {
                 throw new ArgumentNullException(nameof(predefinedConstants));
             }
 
-            _stream = stream;
-            _tokenStream = tokenStream;
-            _symbolTable = symbolTable;
+            _stream = stream ?? throw new ArgumentNullException(nameof(stream));
```

Comment:
As I have already stated elsewhere, I do not really like guard clauses to be hidden away like this.